packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -41,153 +48,160 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
   @State() public error!: Error;
 
+  /**
+   * Number of breadcrumbs to be shown before collapsing.
+   */
   @Prop() public collapseThreshold = 5;
+  /**
+   * Character that divides each path segment in a category facet breadcrumb
+   */
   @Prop() public categoryDivider = '/';
 
+  @BindStateToI18n()
+  @State()
+  public strings: I18nState = {
+    breadcrumb: (variables) =>
+      this.bindings.i18n.t('removeFilterOn', variables),
+  };
+
   public initialize() {
     this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
   }
 
-  private get facetBreadcrumbs() {
-    return this.breadcrumbManagerState.facetBreadcrumbs.map((breadcrumb) => {
-      const breadcrumbsValues = this.getBreadrumbValues(breadcrumb);
-      return (
-        <ul part="breadcrumbs">
-          <li part="breadcrumb-label">{breadcrumb.field}:&nbsp;</li>
-          {breadcrumbsValues}
-        </ul>
-      );
-    });
+  private getBreadcrumbValue(
+    value: string,
+    breadcrumbValue: BreadcrumbValue<BaseFacetValue> | CategoryFacetBreadcrumb
+  ) {
+    return (
+      <button
+        part="breadcrumb-button"
+        class="text-on-background-variant breadcrumb-button flex items-center"
+        aria-label={this.strings.breadcrumb({value})}
+        onClick={() =>
+          this.breadcrumbManager.deselectBreadcrumb(breadcrumbValue)
+        }
+      >
+        <span
+          part="breadcrumb-value-label"
+          class="whitespace-nowrap overflow-ellipsis overflow-hidden"
+        >
+          {value}
+        </span>
+        <div
+          part="breadcrumb-clear"
+          role="button"
+          class="breadcrumb-clear ml-1.5"
+          innerHTML={mainclear}
+        />
+      </button>
+    );
+  }
+
+  private getBreadcrumbWrapper(field: string, children: any) {
+    return (
+      <div class="flex items-center text-sm" part="breadcrumb-wrapper">
+        <span class="text-on-background" part="breadcrumb-label">
+          {facetStore.get(field) || field}:&nbsp;
+        </span>
+        {children}
+      </div>
+    );
   }
 
-  private getBreadrumbValues(breadcrumb: Breadcrumb<FacetValue>) {
+  private getBreadcrumbValues(
+    breadcrumb: Breadcrumb<BaseFacetValue & {value: string}>
+  ) {
     const {breadcrumbsToShow, moreButton} = this.collapsedBreadcrumbsHandler(
       breadcrumb
     );
     const renderedBreadcrumbs = breadcrumbsToShow.map((breadcrumbValue) => (
-      <li part="breadcrumb-value">
-        <button
-          part="breadcrumb-button"
-          aria-label={`Remove inclusion filter on ${breadcrumbValue.value.value}`}
-          onClick={() =>
-            this.breadcrumbManager.deselectBreadcrumb(breadcrumbValue)
-          }
-        >
-          {breadcrumbValue.value.value}
-          {this.mainClear}
-        </button>
+      <li class="breadcrumb" part="breadcrumb">
+        {this.getBreadcrumbValue(breadcrumbValue.value.value, breadcrumbValue)}
```

Comment:
Where does that `value.value.value` pattern comes from ? 
Is it an artifact from headless ?

Because I'm seeing it a lot in this PR...

Reply:
Yeah it comes from headless. It's because we have a Breadcrumb (which represents all the selected values for a facet) which has an array of values (each selected value for the facet). Each element inside that array contains its own `value` which represents the FacetValue/RangeFacetValue object. And the `FacetValue` type has a value attribute (the final string value for the facet).